push_service/app.py

```diff
@@ -323,22 +323,28 @@ def post(self, target_id):  # pylint:disable=no-self-use
         target_id = decode_base64_id(target_id)
         return publish(data, target_id)
 
+# Check for path prefix
+if os.environ.get('PATH_PREFIX'):
```

Comment:
This whole path prefix stuff could be on the top along with other environment variable stuff.